design.md

```
@@ -0,0 +1,149 @@
+    borderColor: "{colors.secondary}"
+    rounded: "{rounded.md}"
+    padding: "28px"
+  input:
+    backgroundColor: "#1F1F1F"
+    borderColor: "{colors.secondary}"
+    textColor: "{colors.neutral-light}"
+    rounded: "{rounded.sm}"
+    padding: "12px 16px"
+---
+
+## Overview
```

markdownlint-cli2 (0.22.1)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)